gatsbyjs/gatsby-starter-hello-world
# Add Gatsby Astronaut placeholder image

Adding this image will allow users to add a relative src to add an image to their page in the Part 1 Tutorial here: https://www.gatsbyjs.org/tutorial/part-one/

See discussion here: https://github.com/gatsbyjs/gatsby/issues/9655

## How to QA

- If you haven't already, globally install the [gatsby-cli](https://www.npmjs.com/package/gatsby-cli); `npm install --global gatsby-cli`
- Run: `gatsby new gatsby-astronaut-pr https://github.com/barrymcgee/gatsby-starter-hello-world/`
- Drop into that folder; `cd gatsby-astronaut-pr`
- Run the starter; `npm start`
- View the site in your browser at; [http://localhost:8000/](http://localhost:8000/)
- You should just see "Hello world"
- Open the site folder in your editor and find; `src/pages/index.js`
- Replace it's contents with;

```
import React from "react"

export default () => (
  <div style={{ color: `purple` }}>
    <h1>Hello Gatsby!</h1>
    <p>What a world.</p>
    <img src="./images/gatsby-astronaut.png" alt="The Gatsby Astronaut illustration" />
  </div>
)

```

Switching back to your browser; [http://localhost:8000/](http://localhost:8000/) should now look like the screenshot below;

<img width="688" alt="screenshot 2018-11-05 at 13 07 07" src="https://user-images.githubusercontent.com/505570/47999921-e0ef6a80-e0fb-11e8-9c0d-411a5c01fea2.png">
